docs/source/pages/Curvature_membrane-only_systems.ipynb

```diff
@@ -0,0 +1,6574 @@
+{
```

Comment:
<p>Could you please make the title more detailed? Imagine you've just been emailed this notebook by someone else with no context.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/membrane-curvature/pull/64/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>